notes-service/src/main/java/org/exoplatform/wiki/service/impl/WikiServiceImpl.java

```diff
@@ -823,6 +823,9 @@ public boolean hasPermissionOnWiki(Wiki wiki, PermissionType permissionType, Ide
 
   @Override
   public boolean hasPermissionOnPage(Page page, PermissionType permissionType, Identity user) throws WikiException {
+    if (page.isDraftPage()) {
+      return page.getAuthor().equals(ConversationState.getCurrent().getIdentity().getUserId());
```

Comment:
why using ConversationState, whil eyou have the user Identity ?

Reply:
Yes you are right.